docs/LEARNING.md

```diff
@@ -0,0 +1,33 @@
+## Learning D
```

Comment:
We don't need titles in these doc files; they get included into pages that we already have titles in.

See https://github.com/exercism/xgo/tree/master/docs for reference.